src/components/common/Logo.vue

```diff
@@ -1,11 +1,36 @@
 <template>
-  <img src="~assets/img/astar_logo_v2.svg" class="icon--logo" />
+  <svg
```

Comment:
Maybe add this to Astar UI in the future

Reply:
@impelcrypto Agree! Let me add this to my task list after dApp staking v3